rkt/image_list.go

```diff
@@ -78,6 +80,54 @@ var (
 	}
 )
 
+type printedImage struct {
+	ID         string `json:"id"`
+	Name       string `json:"name"`
+	ImportTime string `json:"import_time"`
+	LastUsed   string `json:"last_used"`
+	Size       string `json:"size"`
+}
+
+func (r *printedImage) attributes() []string {
+	return []string{r.ID, r.Name, r.Size, r.ImportTime, r.LastUsed}
+}
+
+func (r *printedImage) printableString() string {
+	return strings.Join(r.attributes(), "\t")
+}
+
+type outputFormat int
```

Comment:
I didn't express myself totally clearly on this, but I was also saying some consts could help readability.

Even though go doesn't have real enums, it's sorta idiomatic to make something like them with a type "alias" + consts (sorta like the [iota](https://golang.org/ref/spec#Iota) docs reference). In this case, it would look something like:

```
const (
    outputFormatTabbed outputFormat = iota
    outputFormatJSON
    outputFormatPrettyJSON
)
```

The consts can then be used for assignment and comparison to make things more readable and to give a little bit of type safety.